src/screens/Dashboard/BabyDashboard/BabyDashboard.css

```diff
@@ -0,0 +1,12 @@
+.main-container {
```

Comment:
Notice how you have identical code in 3 different places. `BabyDashboard.css`, `CodingDashboard.css` and `Dashboard.css`. Since all of these files are inside of the `Dashboard` directory, and since all of this css is applied to classnames, you can just remove the css files from Baby and Coding. You can import the single `Dashboard.css` file into all three places.